cps/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -319,7 +319,7 @@ public File getStorageDir() throws IOException {
     public void start() throws IOException {
         final CpsScript s = parseScript();
         scriptClass = s.getClass();
-        s.initialize();
+        s.$initialize();
```

Comment:
I always assumed `$` would have been an invalid char in a java identifier :smile: